change/office-ui-fabric-react-2020-02-27-19-36-04-GroupedListCustomCheckbox.json

```diff
@@ -0,0 +1,9 @@
+{
+  "type": "patch",
+  "comment": "Support custom checkbox in GroupHeader",
```

Comment:
Please polish release note commit comment: "GroupHeader: add support for custom Checkbox."

Reply:
I updated the comment, I hope it is clearer now.

Reply:
I also added an example demonstrating on how to use the new property